google-cloud-error_reporting/acceptance/error_reporting/error_reporting_test.rb

```diff
@@ -52,15 +52,18 @@
     error_group = error_group_stats.first.group
     error_group_id = error_group.group_id
 
-    error_events = nil
+    error_event = nil
     wait_until do
       response = @error_stats_vtk_client.list_events formatted_project,
                                                      error_group_id
       error_events = response.page.response.error_events
-      !error_events.empty?
-    end
 
-    error_event = error_events.first
+      error_events.each do |event|
```

Comment:
Are you looking for the first such event or the last? (Or does it matter?) Unless you're explicitly looking for the last match of potentially multiple matches (and I suspect you aren't), then the following is more clear:

```ruby
error_event = error_events.find{ |event| event.service_context.version == service_version }
```

Reply:
That makes total Ruby sense.